app/src/components/notifications/NotificationBanner.vue

```diff
@@ -51,7 +51,35 @@ switch (props.notification.state) {
 
 <template>
     <div v-if="show" class="inset-x-0 top-0 z-50 text-zinc-900" :class="color">
-        <div class="flex items-center justify-between px-6 py-1">
+        <RouterLink
```

Comment:
Copying all of the markup just to render a `RouterLink` or not will be a bit cumbersome to maintain I think. Instead, I recommend to use a different solution to dynamically render it. This can probably be achieved by using the dynamic `component`, like we do for the icon. `<component :is="notification.routerLink ? RouterLink : 'div'"`

Reply:
@laurensgroeneveld Thanks for your insight. I completely forgot that way to do.